dks-host/dks-host-install.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -x
+
+## Script to install DKS service
+
+# Install Java
+sudo yum update -y
+sudo yum install -y java-1.8.0-openjdk-devel
+
+# Install Amazon SSM agent
+sudo yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
+
+# Download & install latest DKS service artifact
+curl -s https://api.github.com/repos/dwp/data-key-service/releases/latest \
```

Comment:
This is much easier (for me!) to understand as `curl -s https://api.github.com/repos/dwp/data-key-service/releases/latest | jq -r '.assets[0].browser_download_url' | xargs wget --no-check-certificate -O /tmp/dks.jar` but I'm not sure if CentOS installs `jq` by default?

Reply:
No jq there...